docs/webhooks/Receiving-Webhooks.md

```diff
@@ -0,0 +1,28 @@
+---
+tags: [Webhooks]
+---
+
+# Local testing
+
+## Tunnelling traffic to your local machine
+
+To test out webhooks locally without deploying an application to an environment or exposing your test environments to the Internet, you can use ngrok, a free service which tunnels internet traffic to your local environment. 
+
+Download ngrok [here](https://ngrok.com/download), then follow their getting started guide.
+
+## Configuring webhooks through the portal
+
+Once you’re up and running with ngrok, you’ll need to configure our webhooks as follows:
+
+1. Generate a secret in the portal and save its value for later
+2. Create subscription in the portal, using the URL provided by ngrok
```

Comment:
Should we provide examples e.g. `https://a13c-86-154-55-67.ngrok.io/webhooks` to indicate a server endpoint that must be HTTPS.